dali/python/backend_impl.cc

```diff
@@ -70,13 +70,26 @@ static string TensorLayoutRepr(const TensorLayout &tl) {
 }
 
 void ExposeTensorLayout(py::module &m) {
-  py::class_<TensorLayout>(m, "TensorLayout")
+  auto &tl = py::class_<TensorLayout>(m, "TensorLayout")
   .def(py::init([](string s) {
     return new TensorLayout(s);
   }))
   .def("__str__", &TensorLayout::str)
   .def("__repr__", TensorLayoutRepr)
   .def("__len__", &TensorLayout::ndim);
+  #define DEFINE_LAYOUT_CMP(name, expr)\
```

Comment:
move macros to the top of the file

Reply:
It's not a good idea - this only makes sense in this function. What I can do (and will) is to add `#undef` after use.